src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -0,0 +1,71 @@
+interface WebLNNode {
+  alias: string;
+  pubkey?: string;
+  color?: string;
+}
+
+export interface MakeInvoiceArgs {
+  amount: number;
+  memo: string;
+}
+
+export interface MakeInvoiceResponse {
+  data: {
+    paymentRequest: string;
+    rHash: string;
+  };
+}
+
+export interface GetInfoResponse {
+  data: WebLNNode;
+}
+
+export interface GetBalanceResponse {
+  data: {
+    balance: number;
+  };
+}
+
+export type SendPaymentResponse =
+  | {
+      data: {
+        preimage: string;
+        paymentHash: string;
+        route: { total_amt: number; total_fees: number };
```

Comment:
maybe it is a good time to not have this nested in the `route` option? 

originally I tried to follow lnd's response as much as possible, but I am not sure if that makes sense for us